### PR TITLE
[WIP] Python3 support

### PR DESCRIPTION
**Pull request by ccrisan**

### CODE REVIEW

**Comment by cclauss** [comment]
Perhaps add these Travis CI changes as a quick double check:
https://github.com/ccrisan/motioneye/pull/1589/files

File: motioneye/utils.py [hunk 1]
```diff
@@ -275,15 +275,22 @@ def make_str(s):
     if isinstance(s, str):
         return s
 
+    elif isinstance(s, bytes):
```
Comment (cclauss) [inline motioneye/utils.py]: Check out six.ensure_str() and six.ensure_text() https://github.com/benjaminp/six/blob/master/six.py#L901

**Comment by cclauss** [comment]
Three files with conflicts that must be resolved.

**Comment by MichaIng** [comment]
Conflicts resolved.

@marcopaganini
In https://github.com/motioneye-project/motioneye/commit/6a9e709 you changed it so that pip is left installed in the Docker image. Was this intended, i.e. is pip required for something or was it was forgotten? I added it back in for being purged when resolving conflicts, but can be reverted.

**Review by MichaIng — APPROVED** [review]

**Comment by MichaIng** [comment]
Approving, so we can continue with rebasing and merging other Python 3 related PRs and can start testing on the `dev` branch then.

**Comment by MichaIng** [comment]
No worries, it got merged indirectly via #2313, so we are on track.

**Pull request closed without merging**